media/js/firefox/features-landing.js

```diff
@@ -0,0 +1,47 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function() {
+    'use strict';
+
+    var supportsInsersectionObserver = typeof IntersectionObserver !== 'undefined';
+
+    function callback(changes, observer) {
+        changes.forEach(function(change) {
+            if (change.intersectionRatio > 0) {
+                change.target.src = change.target.dataset.src;
+                change.target.onload = function() {
```

Comment:
This is just for the little fade-in, seems a bit nicer than just swapping out the image as it plops in.